### PR TITLE
Endpoint to handle callback from OneLogin

### PR DESCRIPTION
**Pull request by cooperaj**
# Purpose

This endpoint will handle end user requests upon return from One Login. At this stage it will return a dummy /  “hello world” response. This enables testing of the end to end control. 

Fixes UML-3102

## Learning

Going to need a rework of the smoke tests or implementation of alternative ui/acceptance testing for the end to end part.

## Checklist

* [x] I have performed a self-review of my own code
* I have added relevant logging with appropriate levels to my code
* New event_codes have been documented on the [wiki page](https://opgtransform.atlassian.net/wiki/spaces/LSML2/pages/3277881441/Understanding+the+event+logs)
* I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* I have added tests to prove my work
* I have added welsh translation tags and updated translation files
* I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* I have notified the Interaction Designer of any content changes so that appropriate screenshots/flow diagram changes can be made
* The product team have tested these changes

### CODE REVIEW

File: service-front/app/config/routes.php [hunk 1]
```diff
@@ -100,7 +99,7 @@
     $app->get('/session-expired', Actor\Handler\ActorSessionExpiredHandler::class, 'session-expired');
     $app->get('/session-check', Actor\Handler\ActorSessionCheckHandler::class, 'session-check');
     $app->get('/session-refresh', Common\Handler\SessionRefreshHandler::class, 'session-refresh');
-    $app->get('/home/login', Actor\Handler\LoginPageHandler::class, 'auth-redirect');
+    $app->get('/home/login', Actor\Handler\OneLoginCallbackHandler::class, 'auth-redirect');
```
Comment (MishNajam) [inline service-front/app/config/routes.php:102]: Would this need to be behind a feature flag now?

File: Makefile [hunk 2]
```diff
@@ -36,6 +36,12 @@ run:
 	$(COMPOSE) run $(filter-out $@,$(MAKECMDGOALS))
 .PHONY: run
 
+pull:
```
Comment (MishNajam) [inline Makefile:39]: For my own understanding: why was this added 🤔

File: service-front/app/config/routes.php (hunk 1 above)
Comment (cooperaj) [reply service-front/app/config/routes.php:102]: Potentially. But since it's a route that *only* works when accessed as a part of a legitimate flow and does nothing otherwise (or will do, curently it works in that you'll get the hello world) it's not strictly needed. Can add if you'd prefer.

File: Makefile (hunk 2 above)
Comment (cooperaj) [reply Makefile:39]: make build only builds things but since we're now using images hosted on AWS more (read, the one login mock) we need a way to also update those if needed.

File: service-front/app/config/routes.php (hunk 1 above)
Comment (cooperaj) [reply service-front/app/config/routes.php]: I've added a wrapper for the correct feature flag now that 404's if it's not turned on. I've also added some basic sanity checking so it can't be accessed incorrectly

**Review by MishNajam — APPROVED** [review]
